services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -55,6 +63,43 @@ class WriteMetadataActor(override val batchSize: Int,
     dbAction.map(_ => allPutEvents.size)
   }
 
+  private def sanitizeInputs(
+    metadataWriteActions: NonEmptyVector[MetadataWriteAction],
+    metadataKeysToClean: List[String]
+  ): NonEmptyVector[MetadataWriteAction] =
+    metadataWriteActions.map { metadataWriteAction =>
+      val metadataEvents =
+        metadataWriteAction.events.map { event =>
+          if (
+            metadataKeysToClean.contains(event.key.key)
+            || (event.key.key.contains(":")
+              && metadataKeysToClean.contains(event.key.key.substring(0, event.key.key.indexOf(":"))))
```

Comment:
It looks like we're doing something a little more complicated than matching on the keys in the config - we're also matching on keys prefixed with them? Was that necessary for the usage we've seen? 

It might be easier to think about if we did something like this, which will match the full key or a colon-delimited prefix of one or more items:
```
if (metadataKeysToClean.exists(kc => event.key.key == kc  || event.key.key.startsWith(s"${kc}:")) {
    ...
}
```
Either way, we should update the comment in `cromwell.examples.conf` to reflect the prefix behavior.